neo/Prompt/Commands/tests/test_wallet_commands.py

```diff
@@ -875,3 +874,113 @@ def test_wallet_import_multisig_address(self):
             res = CommandWallet().execute(args)
             self.assertTrue(res)
             self.assertIn("Added multi-sig contract address", mock_print.getvalue())
+
+
+class UserWalletSplitTestCase(UserWalletTestCaseBase):
+
+    def test_wallet_split(self):
+        # test wallet split with no wallet open
+        with patch('sys.stdout', new=StringIO()) as mock_print:
+            args = ['split']
+            res = CommandWallet().execute(args)
+            self.assertIsNone(res)
+            self.assertIn("open a wallet", mock_print.getvalue())
+
+        self.OpenWallet1()
+
+        # test wallet split with not enough arguments
+        with patch('sys.stdout', new=StringIO()) as mock_print:
+            args = ['split', self.wallet_1_addr]
+            res = CommandWallet().execute(args)
+            self.assertIsNone(res)
+            self.assertIn("specify the required parameters", mock_print.getvalue())
+
+        # test wallet split with too much arguments
+        with patch('sys.stdout', new=StringIO()) as mock_print:
+            args = ['split', self.wallet_1_addr, 'neo', 0, 2, 'too', 'much']
```

Comment:
We should probably supply all arguments as strings, as that's what the actual code in `execute()` gets to deal with.